src/test/Makefile

```diff
@@ -438,11 +441,24 @@ COMMON_FLAGS = \
 	-O0 \
 	-DUNIT_TEST \
 	-isystem $(GTEST_DIR)/inc \
-	-MMD -MP
+	-MMD -MP \
+	-Wno-c99-extensions \
+	-Wno-reorder
 
 CC_VERSION = $(shell $(CC) -dumpversion)
 CXX_VERSION = $(shell $(CXX) -dumpversion)
 
+# Betaflight supports clang 6 - 10. Please revisit when new version arrive.
+ifeq ($(shell $(CC) -v 2>&1 | grep -c "clang version"), 1)
+    CC_SUPPORTED_VERSION := $(shell expr `clang -dumpversion | cut -f1 -d.` \>= 11)
+    CC_SUPPORTED_VERSION := $(shell expr `clang -dumpversion | cut -f1 -d.` \<= 5)
```

Comment:
This won't quite work, as the second assignment to `CC_SUPPORTED_VERSION` simply overrides and ignores the first one.

Can I propose:
```
CC_MAJOR_VERSION := $(shell clang -dumpversion | cut -f1 -d.)
CC_UNSUPPORTED_VERSION := $(shell expr $(CC_MAJOR_VERSION) \<= 5 | $(CC_MAJOR_VERSION) \>= 11)
```